libdevcrypto/Common.cpp

```diff
@@ -67,6 +70,14 @@ bool dev::SignatureStruct::isValid() const noexcept
 	return true;
 }
 
+bool dev::SignatureStruct::zeroSign() const noexcept
+{
+	if (r == h256(0) && s == h256(0) && v == 0) {
```

Comment:
Style: Don't use braces for single-line body and if you use braces, move them to the next line.